docs/webadministration.rst

```diff
@@ -0,0 +1,437 @@
+
+
+On the new relayed domain page the following options can be entered for a new relayed domain:
+
+* Relayed domain name. The domain name that is relayed. Email messages addressed to this domain (To: John@example.com), will be forwarded to this domain. 
+  No authentication is required. 
+
+* Remote host (optional). The SMPT server that will be used for relaying the email message.
+  When this field is blank, the Mailu server will directly send the email message to the relayed domain.
+  As value can be entered either a hostname or IP address of the SMPT server. 
+  By default port 25 is used. To use a different port append ":port number" to the Remote Host. For example:
+  123.45.67.90:2525.
+
+* Comment. A text field where a comment can be entered to describe the entry.
+
+Changes are effective immediately after clicking the Save button.
+
+NOTE: Due to bug `1588`_ email messages fail to be relayed if no Remote Host is configured. 
+As a workaround the HOSTNAME or IP Address of the SMPT server of the relayed domain can be entered as Remote Host.
+Please note that no MX lookup is performed when entering a hostname as Remote Host. You can use the MX lookup on mxtoolbox.com to find the hostname and IP Address of the SMTP server.
+
+.. _`1588`: https://github.com/Mailu/Mailu/issues/1588
+
+Antispam
+--------
+
+The menu item Antispam opens the Rspamd webgui. For more information how spam filtering works in Mailu see the :ref:`Spam filtering page <antispam_howto_block>`.
+The spam filtering page also contains a section that describes how to create a local blacklist for blocking email messages from specific domains. 
+The Rspamd webgui offers basic functions for setting metric actions, scores, viewing statistics and learning.
+
+The following settings are not persisent and are *lost* when the Antispam container is recreated or restarted:
+
+* On the configuration tab, any changes to config files that do not reside in /var/lib or /etc/rspamd/override.d. The last location is mapped to the Mailu overrides folder.
+
+* All information on the History tab.
+
+
+The following settings are persistent and will survive container recreation:
+
+* All information on the Status tab
+
+* All information on the Throughput tab.
+
+* On the Configuration tab, the changes made to action values (greylist, probably spam ....) and config files that reside in /var/lib or /etc/rspamd/override.d. The last location is mapped to the Mailu overrides folder.
+
+* Any changes made on the Symbols tab.
+
+* Any email messages that have been submitted for spam/ham learning on the Scan/Learn tab.
+
+
+Mail domains
+------------
+
+On the `Mail domains` page all the domains served by Mailu are configured. Via the new domain button (top right) a new mail domain can be added. For existing domains you can access settings via the icons in the Actions column and Manage column. From left to right you have the following options within the Action column and Manage column.
```

Comment:
```suggestion
On the `Mail domains` page all the domains served by Mailu are configured. Via the new domain button (top right) a new mail domain can be added. Please note that you may have to add the new domain to `HOSTNAMES` in your `mailu.env`. For existing domains you can access settings via the icons in the Actions column and Manage column. From left to right you have the following options within the Action column and Manage column.
```

Reply:
Looks good to me. However because it mentions the mailu.env file. The text mailu.env should actually be a hyperlink to the section HOSTNAMES on the configuration reference page. I will try to create a commit for this when I have time..

Reply:
Resolved via commit f843bfe.